src/test/resources/db/user/create-default-test-user.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO users (first_name, last_name, email, password, shipping_address) VALUES("user1", "user1", "user1@email.com", "fakePassword1", "shippingAddress1");
```

Comment:
if you want to test making changes in DB, it's not required to execute SQL scripts for restoring DB after tests, you can use `@Transactional @Rollback` on method annotations or test containers